addons/spreadsheet/static/src/plugins.js

```diff
@@ -0,0 +1,15 @@
+/** @odoo-module */
+
+import { CorePlugin, UIPlugin } from "@odoo/o-spreadsheet";
+
+/**
+ * An o-spreadsheet core plugin with access to all custom Odoo plugins
+ * @type {import("@spreadsheet").OdooCorePluginConstructor}
+ **/
+export const OdooCorePlugin = CorePlugin;
```

Comment:
would be nice if this could be in a type file instead of a js file. Idk if it's feasible tho.

Reply:
I was thinking about leaving CorePlugin in the js code, but add a type anotation to make the typing work. But idk if it's possible, or if it's even better